content/boards/unsupported/raspberry-pi-pico-2/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Raspberry Pi Pico 2
+description: Details on MobiFlight incompatibility with the Raspberry Pi Pico 2
+---
+
+> [!IMPORTANT]
+> This board is unsupported. Consider a [Raspberry Pi Pico (1)](../../raspberry-pi-pico) instead.
```

Comment:
Should be `Raspberry Pi Pico 1` without the parens, to match other references and how they refer to it on their site.